### PR TITLE
BLD: added scaled sigmoid

### PR DESCRIPTION
**Pull request by Jped**

### CODE REVIEW

File: src/matzah.py [hunk 1]
```diff
@@ -122,11 +122,13 @@ def __init__(self, DEVICE, fcSize, numFC, styleDim, outputDim, numBlocks,
         self.Synthesis = SynthesisNetwork(DEVICE, outputDim,
                                           fcSize + numClasses, numBlocks,
                                           startDim, channels, kernel)
+        self.sigmoid = nn.Sigmoid()
 
     def forward(self, styleVec, classVec):
         latentStyle = self.Style.forward(styleVec, classVec)
         # now need to set up the other side of the network.
         generatedCurves = self.Synthesis.forward(latentStyle)
+        generatedCurves = 3 * self.sigmoid(generatedCurves) - 3
```
Comment (eigenfoo) [inline src/matzah.py]: This takes values on the interval [-3, 0]. Are we sure about this?

Comment (ccurro) [reply src/matzah.py]: Yeah this does not look quite right

Comment (Jped) [reply src/matzah.py]: yeah should be 6 * self.sigmoid let me update.

File: src/matzah.py [hunk 2]
```diff
@@ -122,11 +122,13 @@ def __init__(self, DEVICE, fcSize, numFC, styleDim, outputDim, numBlocks,
         self.Synthesis = SynthesisNetwork(DEVICE, outputDim,
                                           fcSize + numClasses, numBlocks,
                                           startDim, channels, kernel)
+        self.sigmoid = nn.Sigmoid()
 
     def forward(self, styleVec, classVec):
         latentStyle = self.Style.forward(styleVec, classVec)
         # now need to set up the other side of the network.
         generatedCurves = self.Synthesis.forward(latentStyle)
+        generatedCurves = 6 * self.sigmoid(generatedCurves) - 3
```
Comment (eigenfoo) [inline src/matzah.py]: Idk if 3 is the number we should targeting here...  https://github.com/ccurro/font-bakers/blob/master/src/utils.py#L43-L46

But this suffices for now; merging.

**Pull request merged**